docs/source/resources/scyllaclusters/storage-resizing.md

```diff
@@ -0,0 +1,99 @@
+# Resizing storage in ScyllaCluster
+
+Due to limitations of the underlying StatefulSet, ScyllaClusters don't allow storage changes. The following procedure describes how to adjust the storage manually.
+
+Let's use the following pods and StatefulSet in the resize examples below.
+   ``` 
+   kubectl get pods                                                      
+                 
+   NAME                                    READY   STATUS    RESTARTS   AGE
+   simple-cluster-us-east-1-us-east-1a-0   2/2     Running   0          12m
+   simple-cluster-us-east-1-us-east-1a-1   2/2     Running   0          11m
+   simple-cluster-us-east-1-us-east-1a-2   2/2     Running   0          9m27s
+    
+   kubectl get statefulset
+   
+   NAME                                  READY   AGE
+   simple-cluster-us-east-1-us-east-1a   3/3     12m
+   ``` 
+
+
+## Increasing the storage size
+
+1. Orphan delete ScyllaCluster
```

Comment:
I think it's important to mention that existing scyllacluster spec needs to be saved somewhere and aligned to new storage requirements. Otherwise someone executing very first step of a procedure may loose the spec and find out later it's no longer available.